subprojects/groovy-sql/src/main/java/groovy/sql/BatchingStatementWrapper.java

```diff
@@ -34,10 +34,10 @@
  * automatically. If batchSize is zero, then no batching is performed.
  */
 public class BatchingStatementWrapper extends GroovyObjectSupport {
-    private Statement delegate;
+    private final Statement delegate;
     protected int batchSize;
     protected int batchCount;
-    protected Logger log;
+    protected final Logger log;
```

Comment:
While it may not be likely, there could be some subclasses that set this so I think it might be good to avoid changing `protected` fields.